backend/src/routers/gameRouter.ts

```diff
@@ -346,9 +358,52 @@ gameRouter.post('/markPlayed', async (req, res) => {
 		res.status(200).json({ message: 'Game marked as played' });
 	} catch (e) {
 		if (e instanceof Error)
-			res.status(400).json({ message: e.message });
+			res.status(500).json({ message: e.message });
+		else
+			res.status(500).json({ message: 'Error marking game as played' });
+	}
+});
+
+/**
+ * @api {post} /api/v1/games/markNotPlayed Saves that a user has played a game
+ * @apiName markPlayed
+ * @apiGroup Games
+ * @apiDescription Marks the game as played for the user
+ *
+ * @apiBody {String} gameId Id of the game
+ * @apiBody {String} userId Id of the user
```

Comment:
Do we really have a userId in the request body? It looks like we are getting it from the authentication system(as we should).